server/views/partials/primaryNav.njk

```diff
@@ -10,9 +10,9 @@
             attributes: { 'data-qa': 'navLinkToDo'}
         },
         {
-            text: 'Find someone',
-            href: '/find-offender',
-            active: isTodoPage != true,
+            text: 'Find a person',
+            href: '/find-person',
+            active: isFindPersonPage == true,
```

Comment:
Can't spot where this is set, I'm guessing WIP as nothing is failing

Reply:
That was inadvertently stashed, have restored thanks